DuckDuckGo/AutoClear.swift

```diff
@@ -73,6 +73,10 @@ class AutoClear {
     private func shouldClearData(elapsedTime: TimeInterval) -> Bool {
         guard let settings = AutoClearSettingsModel(settings: appSettings) else { return false }
 
+        if ProcessInfo.processInfo.arguments.contains("autoclear-ui-test") {
+            return elapsedTime > 5
+        }
+
```

Comment:
When maestro launches with this argument then all expiries are now 5 seconds.  Unit tests and backgrounding UI tests validate that this does not affect production.